### ISSUE TITLE
ENERGY_KILO_WATT_HOUR|POWER_WATT|POWER_KILO_WATT|TEMP_CELSIUS  was used from senec, this is a deprecated constant which will be removed in HA Core 2025.1

### ISSUE THREAD

**Issue by marq24** [marq24/ha-senec-v3#68]
When you see this in your HA log
```
ENERGY_KILO_WATT_HOUR was used from senec, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfEnergy.KILO_WATT_HOUR instead, please create a bug report at https://github.com/marq24/ha-senec-v3/issues
POWER_WATT was used from senec, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfPower.WATT instead, please create a bug report at https://github.com/marq24/ha-senec-v3/issues
POWER_KILO_WATT was used from senec, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfPower.KILO_WATT instead, please create a bug report at https://github.com/marq24/ha-senec-v3/issues
TEMP_CELSIUS was used from senec, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTemperature.CELSIUS instead, please create a bug report at https://github.com/marq24/ha-senec-v3/issues
```

Keep CALM... 2025 is not just around the corner... but I am on it!

**Comment by marq24** [comment]
I am on that - it's a really minor thing - but I am currently working on a larger change that I want to finish before the next update...

**Comment by marq24** [comment]
solved with 2024.0.1
